.github/workflows/tests.yml

```diff
@@ -209,7 +209,14 @@ jobs:
           brew tests ${{ matrix.test-flags }}
         env:
           HOMEBREW_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          HOMEBREW_CODECOV_TOKEN: 3ea0364c-80ce-47a3-9fba-93a940d4b5d7
+
+      - name: Show coverage files
+        run: ls -al /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/test/coverage
+
+      - uses: codecov/codecov-action@e156083f13aff6830c92fc5faa23505779fbf649 # v1.2.1
+        with:
+          directory: /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/test/coverage
```

Comment:
This will only submit test coverage on Linux rather than macOS. Is this intentional/a hard requirement? Note `if RUBY_PLATFORM[/darwin/]` below will stop it from being sent, I think?